pkgs/tools/security/nitrokey-app/default.nix

```diff
@@ -2,13 +2,13 @@
 
 stdenv.mkDerivation rec {
   name = "nitrokey-app";
-  version = "0.5.1";
+  version = "0.6.3";
 
   src = fetchFromGitHub {
     owner = "Nitrokey";
     repo = "nitrokey-app";
     rev = "v${version}";
-    sha256 = "0acb2502r3wa0mry6h8sz1k16zaa4bgnhxwxqd1vd1y42xc6g9bw";
+    sha256 = "1l5l4lwxmyd3jrafw19g12sfc42nd43sv7h7i4krqxnkk6gfx11q";
   };
 
   buildInputs = [
```

Comment:
I fixed the issues you mentioned, thanks for pointing out. 

I keep the patches for now. But I will keep in mind your recommendation and will reexamine my decision when updating to the next upstream version.